README.md

```diff
@@ -22,26 +22,26 @@ A front end library to help you make *​your*​ grass greener than the other s
 You can manually [download our latest release](#latest-release-link), or install with npm/Bower:
 
 ### NPM
-```
+```console
 npm install garden
 ```
 
 ### Bower
-```
+```console
```

Comment:
I didn't like the result of this cause it looks like the same style for links! That's a shame :see_no_evil: 

I'd rather use this style:
```
sudo rm -rf ~/
```
than this one:
```console
sudo rm -rf ~/
```

Reply:
Try ```bash

Reply:
Same style

```bash
test
```

Reply:
I used `console` because there are other pieces of code on this file that use it.

It seems that sh, shell, bash etc are all aliases:
![screen shot 2017-10-24 at 10 12 44 pm](https://user-images.githubusercontent.com/1184252/31974172-b38f5af8-b908-11e7-9977-739ad9fe9425.png)

```sh
npm install garden
```

```sh
bower install garden
```